libraries/Microsoft.Bot.Schema/Teams/TeamsChannelData.cs

```diff
@@ -28,13 +28,15 @@ public TeamsChannelData()
         /// <param name="notification">Notification settings for the message.</param>
         /// <param name="tenant">Information about the tenant in which the
         /// message was sent.</param>
-        public TeamsChannelData(ChannelInfo channel = default, string eventType = default, TeamInfo team = default, NotificationInfo notification = default, TenantInfo tenant = default)
+        /// <param name="onBehalfOf">The OnBehalfOf information of the message.</param>
```

Comment:
I think we'll have issues with API compact here because the signature changed. We should add an overload of the method instead.

Reply:
I added a overload for the constructor